internal/budgetadjustments/events/examples/update_example.sh

```diff
@@ -1,7 +1,7 @@
-# Update Adjustment Evens using a file
+# Update Adjustment Events using a file
```

Comment:
you can delete it since you already have that at line 18 :)

Reply:
rearranged